src/core/public/application/capabilities/capabilities_service.test.ts

```diff
@@ -17,28 +17,30 @@
  * under the License.
  */
 
-import { InjectedMetadataService } from '../../injected_metadata';
+import { httpServiceMock, HttpSetupMock } from '../../http/http_service.mock';
 import { CapabilitiesService } from './capabilities_service';
 import { LegacyApp, App } from '../types';
 
+const mockedCapabilities = {
+  catalogue: {},
+  management: {},
+  navLinks: {
+    app1: true,
+    app2: false,
+    legacyApp1: true,
+    legacyApp2: false,
```

Comment:
is this branch with `capabilities.navLinks[appId] === undefined` covered in tests? https://github.com/restrry/kibana/blob/1625cb34de7e6fddcc391eaeb98a0560d2af1bb9/src/core/public/application/capabilities/capabilities_service.tsx#L51

Reply:
It wasn't actually. Now it does.